resources/lib/dplay.py

```diff
@@ -450,7 +458,7 @@ def get_channels(self):
             url = 'plugin://plugin.video.discoveryplus/?action=play&video_id={channel_id}&video_type=channel'.format(channel_id=value['id'])
 
             channels_list.append(dict(
-                id=key,
+                id='%s@%s'%(key,slugify(xbmcaddon.Addon(id='plugin.video.discoveryplus').getAddonInfo('name'))),
```

Comment:
This breaks epg because this id is used to download channel epg in get_epg(). Can be fixed by adding downloading channels to get_epg() and not to use list returned by get_channels() but this line "epg[channel['id']].append(dict(" also needs slugify or else channel id's doesn't match.

Reply:
I made commit to fix this.